core/src/main/java/hudson/model/View.java

```diff
@@ -1000,6 +1005,41 @@ public synchronized void doDoDelete(StaplerRequest req, StaplerResponse rsp) thr
      */
     public abstract Item doCreateItem( StaplerRequest req, StaplerResponse rsp ) throws IOException, ServletException;
 
+    /**
+     * An API method to get the allowed {$link TopLevelItem}s and its categories.
```

Comment:
Depends on what you mean by “API”. Normally in Jenkins usage this refers to Java APIs unless explicitly qualified by “REST”.